CONTRIBUTING.md

```diff
@@ -312,6 +312,51 @@ load the data. Right now we don't provide any instructions inside here yet, but
 [here's how the official ReasonML playground did
 it](https://github.com/reasonml/reasonml.github.io/blob/source/website/setupSomeArtifacts.js#L65).
 
+### Upgrading the Reason version
+
+The playground is shipping with a specific Reason syntax version, which currently needs to be updated manually once in a while.
+Paths to relevant files:
+
+- `jscomp/main/refmt_api.ml`: Contains the programmatic interface to the Reason syntax converter (responsible for transforming Reason string code to an OCaml AST)
```

Comment:
Let's call it refmt ?

Reply:
`refmt` would indicate it's the binary (at least for me), so I am really into the `Refmt_api` naming

Reply:
Talking about the Reason syntax converter

Reply:
ah, right!